docs/products/mysql/concepts/mysql-replication.rst

```diff
@@ -0,0 +1,31 @@
+MySQL replication
```

Comment:
not sure this file is in the `_toc`?

Reply:
The concepts section do not have all articles specified as you can see here: https://github.com/aiven/devportal/blob/main/_toc.yml#L672

So this is why I do not need to specify this article there

did I understood your question right?